probe/kubernetes/kubelet_test.go

```diff
@@ -0,0 +1,73 @@
+package kubernetes_test
```

Comment:
I will make the symbols private after merging https://github.com/weaveworks/scope/pull/2136 (which contains https://github.com/weaveworks/build-tools/pull/64 )